packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -117,6 +118,16 @@ class ConfigGroup extends GroupHelper {
             const newOptions = configOptions(formattedEnv, argv);
             // When config function returns a promise, resolve it, if not it's resolved by default
             newOptionsObject['options'] = await Promise.resolve(newOptions);
+        } else if (Array.isArray(configOptions) && this.args.configName) {
+            // In case of exporting multiple configurations, If you pass a name to --config-name flag,
+            // webpack will only build that specific configuration.
+            const namedOptions = configOptions.filter((opt) => this.args.configName.includes(opt.name));
+            if (namedOptions.length === 0) {
+                logger.error(`Configuration with name "${this.args.configName}" was not found.`);
+                process.exit(1);
```

Comment:
`0` - fine
`1` - warnings/errors from webpack 
`2` - configuration/options problem or an internal error

Reply:
```suggestion
                process.exit(2);
```

Reply:
+1

Reply:
We need check our code based on this and documented it

Reply:
Somebody can take care?

Reply:
I will take care of this.